projects/agent-test-harness.md

```diff
@@ -0,0 +1,102 @@
+
+## Source Control
+
+The project is hosted in GitHub and includes the following repositories:
+
+* [Agent Test Harness](https://github.com/hyperledger/aries-agent-test-harness) — test results published to [https://aries-interop.info](https://aries-interop.info)
+* [Aries Akrida](https://github.com/hyperledger/aries-akrida) — load testing engine for deployments of ACA-Py and other compatible agents.
+
+## Issue Tracker
+
+Issues are tracked using GitHub's Issues feature in the corresponding repository.
+
+## External Dependencies
+
+The Agent Test Harness itself is built in [Python](https://www.python.org/) using the [Behave](https://github.com/behave/behave) [behavior-driven development](https://en.wikipedia.org/wiki/Behavior-driven_development) (BDD) engine. The repository also contains a component per agent implementation that uses Agent Test Harness.
+
+Test results from the periodic (thrice weekly) interoperability test runs are pushed for later analysis into an instance (operated by [Government of British Columbia](https://www2.gov.bc.ca/gov/content/home)) of the [Allure](https://allurereport.org/docs/pytestbdd/) behavior driven development reporting tool. Those results are in turn published to [https://aries-interop.info](https://aries-interop.info) via a GitHub action in the Agent Test Harness repository.
+
+Akrida is built on [Locust](https://locust.io/) and [Credo-TS](https://github.com/openwallet-foundation/credo-ts) to generate loads applied to any deployment.
+
+## Release Methodology
+
+The only “release” from Agent Test Harness is the test execution results summary published here: [https://aries-interop.info](https://aries-interop.info)
+
+## Initial Maintainers
+
+| Name            | Github      | Organization                             |
+| --------------- | ----------- | ---------------------------------------- |
+| Sheldon Regular | nodlesh     | Funded by Government of British Columbia |
+| Stephen Curran  | swcurran    | Funded by Government of British Columbia |
+| Kim Ebert       | KimEbert42  | Indicio, PBC                             |
+| Alex Walker     | anwalker293 | Inidicio, PBC                            |
+| Ian Costanzo    | ianco       | Funded by Government of British Columbia |
+| Timo Glastra    | TimoGlastra | Animo Solutions                          |
+
+## Proposed Project Governance
+
+The current governance model under Hyperledger is consensus-based. This means that decisions are made through discussions, with the aim of community consensus, as outlined in the [Aries Project Charter](https://docs.google.com/document/d/1F6RbR7xDaBt5CDJhqLJzR4c1pDJtyPGshp9fy6eVtSM/edit?usp=sharing). In cases where no clear consensus is established, a project Technical Steering Committee, or the maintainers (those with escalated GitHub privileges) are granted a louder voice. This approach has proven effective.
```

Comment:
Is the intention for this project to be distinct from the other Aries proposals and their governance processes when it moves to OWF?

Reply:
Agent Test is proposed to be a separate project at OWF, independent of the other artifacts stemming from the existing Hyperledger Aries project, with separate governance. That reflects the distinct purpose of the project (and its sub-projects), and community around it.